symmetric-client/src/main/java/org/jumpmind/symmetric/db/mssql/MsSqlSymmetricDialect.java

```diff
@@ -73,11 +74,20 @@ public MsSqlSymmetricDialect(IParameterService parameterService, IDatabasePlatfo
     public Database readSymmetricSchemaFromXml() {
         Database db = super.readSymmetricSchemaFromXml();
         if (parameterService.is(ParameterConstants.MSSQL_USE_NTYPES_FOR_SYNC)) {
-            Table table = db.findTable(TableConstants.getTableName(getTablePrefix(),
+            Table sym_table = db.findTable(TableConstants.getTableName(getTablePrefix(),
```

Comment:
This should be symTable to follow our lower camel case naming convention.

Reply:
I have fixed it in https://github.com/JumpMind/symmetric-ds/pull/185/commits/69760f23a9004b20e29ec14d17ded246948b6335